spec/features/admin/orders_spec.rb

```diff
@@ -169,6 +169,9 @@ def new_order_with_distribution(distributor, order_cycle)
 
     expect(page).to have_selector 'h1.page-title', text: "Customer Details"
 
+    # The customer selection partial should be visible
+    expect(page).to have_selector '#select-customer'
+
     # And I select that customer's email address and save the order
     targetted_select2_search @customer.email, from: '#customer_search_override', dropdown_css: '.select2-drop'
```

Comment:
Also I'm not sure if `#targetted_select2_search` is behaving as expected, because prior to fixing the view it seems like this select2 element was not actually present on the page, but this part of the spec was not failing. So `#targetted_select2_search` was silently failing to find the element it was looking for, and continuing.